src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.html

```diff
@@ -1,350 +0,0 @@
-              <label
-                class="visEditorSidebar__formLabel"
-                i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.boundsMargin"
-                i18n-default-message="Bounds Margin"
-              ></label>
-              <div class="visEditorSidebar__formControl">
-                <input
-                  name="yBoundsMargin"
-                  class="kuiInput visEditorSidebar__input"
-                  type="number"
-                  step="0.1"
-                  greater-or-equal-than="0"
-                  ng-model="axis.scale.boundsMargin">
-              </div>
-            </div>
-            <div ng-show="axis.scale.boundsMargin < 0">
-              <span
-                class="text-danger"
-                i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.minNeededBoundsMargin"
-                i18n-default-message="Bounds Margin must be greater than or equal to 0"
-              ></span>
-            </div>
-          </div>
-
-          <div class="visEditorSidebar__formRow visEditorSidebar__formRow--expand">
-            <label
-              class="visEditorSidebar__formLabel"
-              for="{ 'setYExtents' + $index }"
-              i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.setAxisExtentsLabel"
-              i18n-default-message="Set Axis Extents"
-            ></label>
-            <div class="visEditorSidebar__formControl">
-              <input class="kuiCheckBox" id="{ 'setYExtents' + $index }" type="checkbox" ng-model="axis.scale.setYExtents" ng-change="updateExtents(axis)">
-            </div>
-          </div>
-
-          <div ng-if="axis.scale.setYExtents">
-            <div class="visEditorSidebar__formRow">
-              <label
-                class="visEditorSidebar__formLabel"
-                i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.maxLabel"
-                i18n-default-message="Max"
-              ></label>
-              <div class="visEditorSidebar__formControl">
-                <input
-                  name="yMax"
-                  class="kuiInput visEditorSidebar__input"
-                  type="number"
-                  step="0.1"
-                  greater-than="axis.scale.min"
-                  ng-model="axis.scale.max"
-                >
-              </div>
-            </div>
-
-            <div class="visEditorSidebar__formRow">
-              <label
-                class="visEditorSidebar__formLabel"
-                i18n-id="kbnVislibVisTypes.controls.pointSeries.valueAxes.minLabel"
-                i18n-default-message="Min"
-              ></label>
-              <div class="visEditorSidebar__formControl">
-                <input
-                  name="yMin"
-                  class="kuiInput visEditorSidebar__input"
-                  type="number"
-                  step="0.1"
-                  greater-than="axis.scale.type === 'log' ? 0 : undefined"
```

Comment:
**note:** moved to https://github.com/elastic/kibana/pull/43772/files?file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-209a462b9cff16d1a72b5e667e14de3bR62